docs/developer_docs/front_end_code.rst

```diff
@@ -89,21 +94,27 @@ To create a new Backbone View, you will either add to an existing Javascript fil
 
 ``this.$el.html(this.template(this.model.attributes));`` this line will insert all the attributes of the model into the template for rendering, and then set the HTML of the subsection of the DOM for the view to the resulting HTML.
 
-For this to work, there must be a file ``kalite/coachreports/hbtemplates/reports/hexagon-counting.handlebars`` that contains the Handlebars.js template for this view. For a simple report, the template might look something like this::
+For this to work, there must be a file ``kalite/coachreports/hbtemplates/reports/hexagon-counting.handlebars`` that contains the Handlebars.js template for this view. For a simple report, the template might look something like this:
+
+.. code-block:: html
 
     <div class="hexagon-report">
         <h3>{{_ "Hexagon Report" }}</h3>
         <p>{{_ "Current number of hexagons:" }}{{number_of_hexagons}}</p>
     </div>
 
-To have this render in a particular Django template, both of the above files would need to be imported through ``<script>`` tags in the Django template. The relevant tags to add in this case would be::
+To have this render in a particular Django template, both of the above files would need to be imported through ``<script>`` tags in the Django template. The relevant tags to add in this case would be:
+
+.. code-block:: html
 
     <script src="{% url 'handlebars_templates' module_name='reports' %}"></script>
     <script type="text/javascript" src="{% static 'js/coachreports/hexagon_report.js' %}"></script>
```

Comment:
Also not recommended, now the dev should create a new bundle for the page, or add to an existing one.